src/python/WMCore/WMSpec/Steps/Executors/LogArchive.py

```diff
@@ -58,9 +58,19 @@ def execute(self, emulator=None, **overrides):
             return emulator.emulate(self.step, self.job)
 
         overrides = {}
+        #TODO need to set override using addOverride method in WMStep
```

Comment:
Still need to set up the override value from spec definition. 
and set it using 
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/WMSpec/WMStep.py#L139